app-doc/zeal/zeal-0.6.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils cmake-utils xdg-utils
```

Comment:
You don't need gnome2-utils since the functions you use here have been migrated to xdg or xdg-utils eclass (and you already inherit xdg-utils). This should allow EAPI-7 bump too.

Update pkg_ phases to use xdg instead.